lua/telescope/_extensions/zoxide/config.lua

```diff
@@ -14,9 +14,6 @@ local default_config = {
       action = function(selection)
         vim.cmd("cd " .. selection.path)
       end,
-      after_action = function(selection)
-        print("Directory changed to " .. selection.path)
```

Comment:
The main reason I added this, is to have a clear default with optionally overriding it (see [overridable-config](https://github.com/jvgrootveld/telescope-zoxide#overridable-config) I could understand some find it annoying, but I don't know what the majority wants as a default.

Anyone could just override it with an empty function. What do you think?
1. Have a print by default where people can optionally override it with an empty function (maybe add it as a suggestion in de Readme)
2. Have an empty function by default where people can optionally override it with a print function (example already in the Readme)

PS. On a side note, although small, this changes the default behaviour of the current user-base.

Reply:
Your plugin your rules, very happy to just override it in my custom config :) My take is that

1) showing sucessfully achieving the user intended behavior isn't something that warants an alert - that's the default expectation. Alerts should be reserved for non-default expectation.
2) Showing the current working directory change is solved through the status line and doesn't appear the responsibility of a switching directory
3) if you use zoxide in the terminal, it doesn't print print the new directory either, so this is inconsistent with zoxide. Equally for cd

Now as a counter argument against myself

vim's `cd`, which this emulates, also appears to print the new directory. I don't like that for the same reasons but it would be inconsistent with it.

Again, I'll take your guidance here - whatever you prefer.